src/Directory.Packages.props

```diff
@@ -20,6 +20,7 @@
     <PackageVersion Include="Newtonsoft.Json" Version="13.0.1" />
     <PackageVersion Include="NUnit" Version="3.13.3" />
     <PackageVersion Include="NUnit3TestAdapter" Version="4.3.0-alpha-net7.4" />
+    <PackageVersion Include="Uno.UI.RuntimeTests.Engine" Version="0.1.0-dev.32" />
```

Comment:
waiting on unoplatform/uno.ui.runtimetests.engine#59
and need to update package